autospec/autospec.py

```diff
@@ -116,6 +117,17 @@ def read_old_metadata():
             archives)
 
 
+def save_build_log(path, iteration):
+    """
+    Save build log to <path>/results/build.log.<iteration>
+
+    Must be saved outside of the results/ directory since it gets wiped away on
+    each round.
```

Comment:
It looks like the log is saved to a new file within the results directory, if I'm reading the code correctly? So is this comment out of date?

Reply:
Out of date comment.

Reply:
Resolved. And actually you were mistaken about the code. The comment on line 122 was out of date, not the comment on 124. The `<path>/results/build.log` is copied to `<path>/build.log.round<iteration>`

Reply:
Ah, right. I think I did misread the code. Thanks.